From d83dec996f88f7d66267b7195ac4ac5511d283b1 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 10 Jul 2017 22:09:07 +0800
Subject: [PATCH 2088/5242] MLK-15933 arm64: dts: freescale: imx8qm: update
 A72 cpu freq table

commit  478ea9f62d00258004a66509d6b3746b8eef289d from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8QM SCFW fixes the HP PLL rate calculation, A72 cluster
CPU frequency has been changed from 1584MHz to 1596MHz,
change the CPU OPP table accordingly.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 15139f5..ada2637 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -2367,7 +2367,7 @@
 &A72_0 {
 	operating-points = <
 		/* kHz    uV */
-		1584000 1150000
+		1596000 1150000
 	>;
 	clocks = <&clk IMX8QM_A72_DIV>;
 };
-- 
1.7.9.5

